packages/neptune-extension/src/api-spec/leaderboard-swagger.json

```diff
@@ -28,16 +28,33 @@
         "operationId" : "getLeaderboard",
         "produces" : [ "application/json" ],
         "parameters" : [ {
+          "name" : "checkpointId",
```

Comment:
Whole content of `leaderboard-swagger.json` is taken from master of main repo.
Nothing changed manually.